src/components/data/kanban/kanban.tsx

```diff
@@ -220,359 +124,207 @@ export const Kanban: React.FC<KanbanProps> = ({
         </Body>
       )}
       <Body className="mykn-kanban__body">
-        <Grid cols={fieldsetsState.length}>
-          {fieldsetsState.map((fieldset, index) => (
-            <KanbanSection
-              key={fieldset[0] ? fieldset[0] : index}
-              columnIndex={index}
-              buttonLinkProps={buttonLinkProps}
-              buttonProps={buttonProps}
-              draggable={draggable}
-              dragIndex={dragIndexState}
-              fieldset={fieldset}
-              fieldsets={fieldsetsState}
-              fieldsetIndex={index}
-              labelSelectColumn={labelSelectColumn}
-              labelMoveObject={labelMoveObject}
-              objectList={objectListsState[index]}
-              renderPreview={renderPreview}
-              urlFields={urlFields}
-              onClick={onClick}
-              onDragOver={handleDragOver}
-              onDrop={handleDrop}
-              onObjectChange={handleObjectChange}
-            />
-          ))}
+        <Grid cols={componentListState.length}>
+          {componentListState.map((column, columnIndex) => {
+            const count = column.items.length;
+            return (
+              <div key={columnIndex}>
```

Comment:
If the column has some unique property, maybe that could be used instead as the `key`?

Reply:
Noted